core/src/main/java/org/sonarsource/sonarlint/core/container/global/ExtensionUtils.java

```diff
@@ -37,8 +39,14 @@ public static boolean isInstantiationStrategy(Object extension, String strategy)
     return InstantiationStrategy.PER_PROJECT.equals(strategy);
   }
 
-  public static boolean isBatchSide(Object extension) {
-    return AnnotationUtils.getAnnotation(extension, BatchSide.class) != null;
+  public static boolean isSonarLintSide(Object extension) {
+    return AnnotationUtils.getAnnotation(extension, SonarLintSide.class) != null;
+  }
+
+  public static boolean isScannerSide(Object extension) {
+    return AnnotationUtils.getAnnotation(extension, BatchSide.class) != null ||
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "BatchSide"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)